doc/book/migration/to-v3-0.md

```diff
@@ -412,22 +412,22 @@ the section above on [dependency reduction](#dependency-reduction).
 
 ## Zend\Mvc\View\InjectTemplateListener
 
-The `InjectTemplateListener` attempts to map a controller *service* name to a
+The `InjectTemplateListener` attempts to map a controller name to a
 template using a variety of heuristics, including an explicit map provided
-during configuration, or auto-detection based on the service name. 
+during configuration, or auto-detection based on the controller class name.
 
 In version 2, the autodetection took into consideration the `__NAMESPACE__`
-provided in routing configuration, and would omit the module subnamespace if a
-match was found. This caused issues when multiple modules shared a top-level
-namespace (e.g., `ZF\Apigility` and `ZF\Apigility\Admin`) if each had a
-controller with the same name.
+route match parameter to derive subnamespaces or would omit them completely if
+`__NAMESPACE__` is not present. This caused issues when multiple modules shared
+a top-level namespace (e.g., `ZF\Apigility` and `ZF\Apigility\Admin`) if each
+had a controller with the same name.
 
 To avoid naming conflicts, version 3 removes this aspect of autodetection, and
 instead provides exactly one workflow for mapping:
 
 - Strip the `Controller` subnamespace, if present (e.g.,
-  the namespace `Application\Controller\\` is normalized to
-  `Application\\`).
+  the namespace `Application\Controller\` is normalized to
+  `Application\`).
```

Comment:
I noticed that github markdown renderer does not treat backslash as special character in backticked strings and docs site appears to be using same renderer:
https://zendframework.github.io/zend-mvc/migration/to-v3-0/#zendmvcviewinjecttemplatelistener